src/Microsoft.Identity.Web.GraphServiceClient/Microsoft.Identity.Web.GraphServiceClient.csproj

```diff
@@ -20,7 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Graph" Version="5.40.0" />
+    <PackageReference Include="Microsoft.Graph" Version="5.51.0" />
```

Comment:
Updated to this version because the transitive dependency ([Microsoft.Kiota.Http.HttpClientLibrary](https://www.nuget.org/packages/Microsoft.Kiota.Http.HttpClientLibrary/1.3.4#dependencies-body-tab)) for the previous version doesn't support System.Text.Json 9.0.0. This is the minimum Graph version that does. 

Alternative is to include 5.51.0 on .NET 9 and 5.40.0 on other targets, but I think this is more consistent.